vars.tf

```diff
@@ -0,0 +1,106 @@
+variable "region" {
+  default = "us-west2"
+}
+
+variable "availability_zone" {
+  description = "The zone where the cluster will be deployed [a,b,...]"
+  default     = ["a", "b"]
+  type        = list(string)
+}
+
+variable "instance_group_name" {
+  description = "The name of the GCP instance group"
+  default     = "redpanda-group"
+}
+
+variable "broker_count" {
+  description = "The number of Redpanda brokers to deploy."
+  type        = number
+  default     = "3"
+}
+
+variable "ha" {
+  description = "Whether to use placement groups to create an HA topology"
+  type        = bool
+  default     = false
+}
+
+variable "client_nodes" {
+  description = "The number of clients to deploy."
+  type        = number
+  default     = "1"
+}
+
+variable "disks" {
+  description = "The number of local disks on each machine."
+  type        = number
+  default     = 1
+}
+
+variable "image" {
+  # See https://cloud.google.com/compute/docs/images#os-compute-support
+  # for an updated list.
+  default = "ubuntu-os-cloud/ubuntu-2004-lts"
+}
+
+variable machine_type {
+  # List of available machines per region/ zone:
+  # https://cloud.google.com/compute/docs/regions-zones#available
+  default = "n2-standard-2"
+}
+
+variable monitor_machine_type {
+  default = "n2-standard-2"
+}
+
+variable client_machine_type {
+  default = "n2-standard-2"
+}
+
+variable "public_key_path" {
+  description = "The ssh key."
```

Comment:
Why not using a default like https://github.com/redpanda-data/terraform-aws-redpanda-cluster/blob/main/vars.tf#L165 ?

Reply:
Done